src/cdef_apply.rs

```diff
@@ -196,7 +196,7 @@ pub(crate) unsafe fn rav1d_cdef_brow<BD: BitDepth>(
     let uv_dir: &[u8; 8] = &UV_DIRS[(layout == Rav1dPixelLayout::I422) as usize];
 
     let have_tt = c.tc.len() > 1;
-    let sb128 = f.seq_hdr.as_ref().unwrap().sb128;
+    let sb128 = f.seq_hdr.as_ref().unwrap().sb128 as c_int;
```

Comment:
```suggestion
    let sb128 = f.seq_hdr.as_ref().unwrap().sb128;
```

Can you move the `as c_int` casts to where it's needed?